build.gradle

```diff
@@ -38,7 +38,7 @@ subprojects {
 
   apply plugin: 'net.ltgt.errorprone'
   configurations.errorprone {
-    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.11'
+    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.5'
```

Comment:
I believe I already have a fix for that, I must have forgotten to push it to the public repo on GitHub.